src/ucp/core/ucp_ep.h

```diff
@@ -345,6 +345,10 @@ typedef struct ucp_wireup_client_data {
 typedef struct ucp_conn_request {
     ucp_listener_h              listener;
     uct_conn_request_h          uct_req;
+    uint8_t                     wiface_idx;  /**< Index of the listening wiface in
```

Comment:
keep index on the worker as ucp_rsc_index_t

Reply:
was decided to cancel